cfgov/teachers_digital_platform/views.py

```diff
@@ -12,15 +12,34 @@
 
 from formtools.wizard.views import NamedUrlCookieWizardView
 
+from wagtail.core.models import Page
+
 from core.decorators import akamai_no_store
 
+from v1.models import SublandingPage
+
+from .forms import SharedUrlForm
 from .resultsContent import ResultsContent
 from .surveys import AVAILABLE_SURVEYS, ChoiceList, Question, get_survey
 from .UrlEncoder import UrlEncoder
 
 
 _tdp = 'teachers_digital_platform'
-_signer = signing.Signer()
+_gradeSelectionPagePk = 15596
+
+
+def _findSurveyGradeSelectionUrl(request: Optional[HttpRequest],
+                                 default='../../../assess/survey/'):
+    """
+    Get URL of the survey grade selection page from Wagtail
+    """
+    try:
+        destination_page = SublandingPage.objects.get(pk=_gradeSelectionPagePk)
+    except Page.DoesNotExist:
```

Comment:
`DoesNotExist` is actually available on all Django models, so you could catch `SublandingPage.DoesNotExist` here and you won't need the `Page` import.